index.html

```diff
@@ -7,20 +7,230 @@
+              <a href="about.html" class="nav-li">About Us</a>
+            </li>
+          </ul>
+          <div class="hamburger-bar">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </div>
+
+        <!-- mobile nav display -->
+        <div class="container-nav">
+          <ul class="mobile-nav-ul">
+            <li class="mobile-nav-li">
+              <a href="#dish-section" class="mobile-nav-li">Dishes</a>
+            </li>
+            <li class="mobile-nav-li">
+              <a href="#reviews" class="mobile-nav-li">Comments</a>
+            </li>
+            <li class="mobile-nav-li">
+              <a href="#partners" class="mobile-nav-li">Partners</a>
+            </li>
+            <li class="mobile-nav-li">
+              <a href="about.html" class="mobile-nav-li">About Us</a>
+            </li>
+          </ul>
+
+          <ul class="hamb-ul">
+            <li class="close-hamb">&times;</li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section id="intro" class="fade-in">
+        <div class="flex-contain">
+          <div class="intro-container">
+            <p class="introP">"Good food guarantees healthy living."</p>
+            <h1 class="intro-header">
+              Get your favorite meals delivered to You!
+            </h1>
+
+            <form class="my-form">
+              <div class="address-input">
+                <label for="address"
+                  >Where do you want your meal delivered?</label
+                >
+                <input
+                  type="text"
+                  class="name-input"
+                  id="address"
+                  name="address"
+                  placeholder="Enter your address"
+                />
+              </div>
+              <div class="time-input">
+                <label for="date-and-time"
+                  >When do you want your meal delivered?</label
+                >
+                <input
+                  type="datetime-local"
+                  id="date-and-time"
+                  name="date-and-time"
+                  class="name-input"
+                  placeholder="input date"
+                  value="2022-04-24T08:44"
+                />
+              </div>
+              <div class="btn-container">
+                <button class="find-food" type="submit">Find Food</button>
+              </div>
+              <p class="small-text">
+                <a href="" class="sign-in-sm">sign in</a>
+                for recent address
+              </p>
+            </form>
+            <p class="chopP">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+              Architecto modi facere rerum sequi veritatis iusto enim.
+            </p>
+          </div>
+        </div>
+      </section>
+      <!-- dishes section -->
+      <section id="dish-section">
```

Comment:
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible, add all alt tags in this section 👍🏿